SU-SOE/stanford_soe_helper
# Soeop 101

# READY FOR REVIEW

# Summary
- Fix spotlight sorting functionality

# Needed By (Date)
- 10/7

# Urgency
- critical

# Steps to Test
- Make sure that sorting does not break on spotlight page when on a next page. See Ticket.

# Affected Projects or Products
- soe main site

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/SOEOP-101
